### DISCUSSION TITLE
index

### DISCUSSION THREAD

**Discussion by giscus** [jzero-io/jzero#31] [Announcements]
# index

特性 企业级代码规范 支持自定义模板, 基于模板新建项目和生成代码, 默认支持多场景开发模板 一键创建项目, 一键生成服务端代码, 数据库代码, 客户端代码 基于 go-zero 框架, 扩展 go-zero 功能，并能完全同步 go-zero 新特性 快速开始 Windows 用户请在 powershell 下执行所有指令

http://localhost:8080/

**Comment by jaronnie** [comment]
第一个评论！

**Comment by xiaoxiaoye** [comment]
不错，挺好用的
